rfd/0037-desktop-access-protocol.md

```diff
@@ -200,11 +210,28 @@ This message contains a mouse wheel update. Sent from client to server.
 #### 9 - error
 
 ```
-| message type (9) | message_length uint32 | message []byte
+| message type (9) | message_length uint32 | message []byte |
 ```
 
 This message indicates an error has occurred.
 
+#### 28 - notification
+
+```
+| message type (28) | message_length uint32 | message []byte | severity byte
+```
+
+This message sends a notification message with a severity level.
+
+`severity` defines the severity of the `message`:
+- `0` is for an error
```

Comment:
0 for error just seems odd. 0 indicates success in almost every protocol/API I can think of. Can  we skip over 0 and make 1 error, 2 warning, 3 informational ?

Reply:
I don't want to skip 0 because we use it in other enums i.e. in the mouse wheel scroll message
```
`axis` identifies which axis the scroll happened on:
- `0` is vertical scroll
- `1` is horizontal scroll
```

How about 0 info, 1 warning, 2 error?